gbmodel/model_sqlalchemy.py

```diff
@@ -147,13 +148,14 @@ def remove_student(self, sts, t_name, session_id):
             return False
         removed_student = removed_students()
         params = {'name': t_name, 'session_id': session_id}
-        result = engine.execute("select id from teams where name = :name AND session_id = :session_id", params)
+        result = engine.execute("select id from teams where name = :name AND session_id = :session_id",
+                                params)
         id = result.fetchone()
         tid = id[0]
 
         for i in sts:
             params = {'name': i, 'tid': tid, 'session_id': session_id}
-            result = engine.execute("select id, tid, session_id, name, is_lead, midterm_done, final_done from students where name = :name AND tid= :tid AND session_id = :session_id", params)
+            result = engine.execute("select id, tid, session_id, name, is_lead, midterm_done, final_done from students where name = :name AND tid= :tid AND session_id = :session_id", params)  # noqa
```

Comment:
Longer than 110 characters?

Reply:
Since we're switching to the ORM anyway I'm leaving this in and silencing it for now, then will fix in a separate PR.

Reply:
(the `# noqa` bit at the end of the line makes flake8, mypy, and presumably other linters ignore the line)